conf/peers.xml

```diff
@@ -44,6 +44,9 @@
  	<!-- you can specify an even rtp port to use for incoming and outgoing rtp
  	     traffic, 0 can be used to choose a random even free port -->
   <rtpPort>0</rtpPort>
+  <!-- if you need to specify an authorization username different than
+       the userPart -->
```

Comment:
in this comment, could you add the default behavior if this parameter is empty (i.e. reuse value from <userPart> tag)? thanks